types/jsforce/record.d.ts

```diff
@@ -1,6 +1,16 @@
+import { RecordResult } from './record-result';
+import { Connection } from './connection';
 import { SalesforceId } from './salesforce-id';
+import { Stream } from 'stream';
 
-export interface Record {
-    Id: SalesforceId;
-    attributes: Object[];
+export class RecordReference<T> {
```

Comment:
http://jsforce.github.io/jsforce/doc/RecordReference.html

Reply:
good job, not sure how i missed this one